lib/Doctrine/Common/Collections/ArrayCollection.php

```diff
@@ -44,7 +45,7 @@
  * @template-implements Selectable<TKey,T>
  * @psalm-consistent-constructor
  */
-class ArrayCollection implements Collection, Selectable
+class ArrayCollection implements Collection, Selectable, Stringable
```

Comment:
I wasn't aware that `ArrayCollection`s could be cast to string and I must admit, I find the implementation rather surprising. 😅